src/Jackalope/Transport/AbstractReadLoggingWrapper.php

```diff
@@ -191,7 +191,7 @@ public function setNodeTypeManager($nodeTypeManager)
     /**
      * {@inheritDoc}
      */
-    public function getNodeTypes($nodeTypes = array())
+    public function getNodeTypes(array $nodeTypes = array())
```

Comment:
i had to revert this change to the interface you did in the first PR, as its a BC break for the implementations. you need to remove it here too.